src/riverroute/RtmIO.F90

```diff
@@ -176,13 +176,22 @@ subroutine ncd_pio_openfile(file, fname, mode)
     character(len=*),parameter :: subname='ncd_pio_openfile' ! subroutine name
     !-----------------------------------------------------------------------
 
-    ierr = pio_openfile(pio_subsystem, file, io_type, fname, mode)
+    if ( fname(1:1) == " " )then
```

Comment:
@slevis-lmwg these are some error messaging changes I made that helped me with this work, and will help in the future. This might be good for you to look over.

Reply:
We talked about this a bit and figured it's fine as is.